### ISSUE TITLE
I am a student of Tsinghua University, doing research in Tencent. When I train with yolov5, the following problems appear，Sincerely hope to get help,

### ISSUE THREAD

**Issue by xianglei3** [ultralytics/yolov5#2203]
My environment was win10 Python 3.7 Pytorch 1.7
I use the yolov5s model 
I use the latest version of 2021.2.13 official websitend the data set voc2014
My CUDA was version 10.2

When I run Python train.py , The error is as follows
RuntimeError: ZeroDivisionError
 File "E:/yoloV5/yolov5-master/train.py", line 83, in train
    model = Model(opt.cfg or ckpt['model'].yaml, ch=3, nc=nc).to(device)  # create
  File "C:\Users\asus\anaconda3\envs\test\lib\site-packages\torch\nn\modules\module.py", line 778, in __getattr__
    raise ModuleAttributeError("'{}' object has no attribute '{}'".format(
torch.nn.modules.module.ModuleAttributeError: 'Model' object has no attribute 'yaml'

The error comes from the following code:
ckpt = torch.load(weights, map_location=device)  # load checkpoint
        if hyp.get('anchors'):
            ckpt['model'].yaml['anchors'] = round(hyp['anchors'])  # force autoanchor
        model = Model(opt.cfg or ckpt['model'].yaml, ch=3, nc=nc).to(device)  # create

The key question is “Model' object has no attribute 'yaml'”

I found two important clue ：
first :
I found that “opt.cfg” was empty and could not print anything, the key codes are as follows
print (opt.cfg)
model = Model(opt.cfg or ckpt['model'].yaml, ch=3, nc=nc).to(device)  # create

second:
In the following code, the program runs the else branch，so  ckpt['model'].yaml is not properly initialized or initialized，the key codes are as follows
    if pretrained:
        with torch_distributed_zero_first(rank):
            attempt_download(weights)  # download if not found locally
        ckpt = torch.load(weights, map_location=device)  # load checkpoint
        if hyp.get('anchors'):
            ckpt['model'].yaml['anchors'] = round(hyp['anchors'])  # force autoanchor

To sum up, when I was training, the error of the console printing was：
    model = Model(opt.cfg or ckpt['model'].yaml, ch=3, nc=nc).to(device)  # create
  File "C:\Users\asus\anaconda3\envs\test\lib\site-packages\torch\nn\modules\module.py", line 778, in __getattr__
    raise ModuleAttributeError("'{}' object has no attribute '{}'".format(
torch.nn.modules.module.ModuleAttributeError: 'Model' object has no attribute 'yaml'

I look forward to your help. Thank you very much. Thank you forever

**Comment by xianglei3** [comment]
the code is  
    if pretrained:
        with torch_distributed_zero_first(rank):
            attempt_download(weights)  # download if not found locally
        ckpt = torch.load(weights, map_location=device)  # load checkpoint
        if hyp.get('anchors'):
            ckpt['model'].yaml['anchors'] = round(hyp['anchors'])  # force autoanchor

**Comment by glenn-jocher** [comment]
@xianglei3 VOC trains very well with YOLOv5, >0.90 mAP. You may be using an older or corrupted model. YOLOv5 will auto-download new models on demand the first time you request them, so I would simply start from scratch with a new git clone, and follow the VOC training commands shown in the Colab Notebook (Appendix section):
https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb

```python
# VOC
for b, m in zip([64, 48, 32, 16], ['yolov5s', 'yolov5m', 'yolov5l', 'yolov5x']):  # zip(batch_size, model)
  !python train.py --batch {b} --weights {m}.pt --data voc.yaml --epochs 50 --cache --img 512 --nosave --hyp hyp.finetune.yaml --project VOC --name {m}
```

If you just want to train say YOLOv5m on VOC, you can run this command, no need to download VOC or download any pretrained weights, it's all done automatically:
```python
python train.py --batch 48 --weights yolov5m.pt --data voc.yaml --epochs 50 --cache --img 512 --nosave --hyp hyp.finetune.yaml --project VOC --name yolov5m
```
